src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -1071,80 +1070,77 @@ def make_integer_prop(
     return prop_
 
 
-def make_interval_prop(
-    name: str,
-    value: Union[PropertyElement, str, Iterable[Union[PropertyElement, str]]],
-    calling_resource: str = "",
-) -> etree._Element:
+def make_interval_prop(name: str, value: Union[PropertyElement, str], calling_resource: str = "") -> etree._Element:
     """
-    Make a <interval-prop> from one or more DSP intervals. The interval(s) can be provided as string or as
-    PropertyElement with a string inside. If provided as string, the permissions default to "prop-default".
+    Make a <interval-prop name="hasSegmentBounds"> for a <video-segment> or <audio-segment>.
+    The interval can be provided as string or as PropertyElement with a string inside.
+    If provided as string, the permissions default to "prop-default".
+    DSP interval values are formatted as "start_seconds:end_seconds".
+    Both numbers can have a decimal point, for fractions of seconds.
 
     Args:
-        name: the name of this property as defined in the onto
-        value: one or more DSP intervals, as string/PropertyElement, or as iterable of strings/PropertyElements
+        name: the name of this property. The only accepted value is "hasSegmentBounds"
+        value: a DSP interval, as string or PropertyElement
         calling_resource: the name of the parent resource (for better error messages)
 
     Warns:
-        If the value is not a valid DSP interval
+        - If the value is not a valid DSP interval
+        - If the name is not "hasSegmentBounds"
 
     Returns:
         an etree._Element that can be appended to the parent resource with resource.append(make_*_prop(...))
 
     Examples:
-        >>> excel2xml.make_interval_prop(":testproperty", "61:3600")
-                <interval-prop name=":testproperty">
-                    <interval permissions="prop-default">61:3600</interval>
+        >>> interval = excel3xml.create_interval_value(start="0:01:00", end="0:02:00")  # result: "60:120"
```

Comment:
not excel3xml

Reply:
Haha, that's a funny one. Thanks for spotting it 🤣